src/main/java/com/group24/decide/LIC.java

```diff
@@ -213,8 +213,27 @@ public boolean Condition8(){
      * @return Return true if the condition is met, other
      * returns false.
      */
-    public static boolean Condition9(){
```

Comment:
You could add a short description here what is checked with this Condtion, because that's what will end up in our Javadocs